msal/tests/Test.MSAL.NET.UIAutomation/MSALMobileTestHelper.cs

```diff
@@ -309,5 +325,21 @@ public void PerformB2CSignInFlow(ITestController controller, LabUser user, B2CId
             }
             CoreMobileTestHelper.VerifyResult(controller);
         }
+
+        public void PerformB2CSignInEditProfileFlow(ITestController controller, B2CIdentityProvider b2CIdentityProvider)
+        {
+            SetB2CInputDataForEditProfileAuthority(controller);
+            
+            controller.Tap(CoreUiTestConstants.AcquirePageID);
+            
+            SetUiBehavior(controller, CoreUiTestConstants.UIBehaviorNone);
+
+            //Acquire token flow
+            controller.Tap(CoreUiTestConstants.AcquireTokenID);
+
+            controller.Tap(CoreUiTestConstants.B2CEditProfileContinueID, XamarinSelector.ByHtmlIdAttribute);
```

Comment:
I guessed on the name of the id and got it. lol. 😹

Reply:
:D